src/dali/abstract_ccxt_input_plugin.py

```diff
@@ -147,8 +147,12 @@ def _get_markets(self) -> List[str]:
 
     @staticmethod
     def _rp2_timestamp_from_ms_epoch(epoch_timestamp: str) -> str:
-        rp2_time = datetime.fromtimestamp((int(epoch_timestamp) / _MS_IN_SECOND), timezone.utc)
+        epoch_seconds, dummy_epoch_ms = divmod(float(epoch_timestamp), _MS_IN_SECOND)
```

Comment:
You could make this a lot simpler by just using integer divison:
```
epoch_in_seconds: int = int(epoch_timestamp) // _MS_IN_SECOND
```
This will provide the same accuracy and easier to read IMHO.